autogen.sh

```diff
@@ -8,9 +8,9 @@ Darwin)
         if [ -d $homebrew_aclocal ]; then
           ACLOCAL_ARGS="$ACLOCAL_ARGS -I $homebrew_aclocal"
         fi
-        gettext_aclocal="$(echo /usr/local/Cellar/gettext/*/share/aclocal)"
-        if [ -d $gettext_aclocal ]; then
-          ACLOCAL_ARGS="$ACLOCAL_ARGS -I $gettext_aclocal"
+        gettext_aclocal="$(echo /usr/local/Cellar/gettext/*/share/aclocal | awk '{ print $NF }')"
+        if [ -d "$gettext_aclocal" ]; then
+          $ACLOCAL_ARGS="$ACLOCAL_ARGS -I $gettext_aclocal"
         fi
```

Comment:
It seems that `ACLOCAL_ARGS` must not be expanded in this context.

It should be:
```
ACLOCAL_ARGS="$ACLOCAL_ARGS -I $gettext_aclocal"
```